frontend/src/pages/index.module.scss

```
@@ -79,21 +79,40 @@
   .steps {
     display: flex;
     flex-direction: column;
-    gap: $spacing-2xl;
+    gap: $layout-xl;
     list-style-type: none;
-    margin: 0;
     padding: $spacing-2xl 0;
-    text-align: center;
+    text-align: left;
```

I don't know if we support right-to-left scripts, but still a good habit to form:

```suggestion
    text-align: start;
```

---

Oh! Yes! Good callout